### ISSUE TITLE
Make copy and move notes action consistent with operating system

### ISSUE THREAD

**Issue by laurent22** [laurent22/joplin#10550]
### Operating system

macOS

### Joplin version

3.0.6

### Current behaviour

Ref: https://discourse.joplinapp.org/t/cannot-easily-copy-notes/38472

- When dragging and dropping a note to a notebook, the displayed cursor icon should be a "move" icon (or no icon, but not  a "+" icon). The action should be a move.
- When using the keyboard modifier (Opt), the action should become a copy. The cursor should change to a "+"

The logic should be similar for Windows and Linux but not sure what the keyboard modifier should be. Maybe "alt"?

**Comment by personalizedrefrigerator** [comment]
While working on this, we should be sure to check that https://github.com/laurent22/joplin/issues/8370 doesn't become an issue again.

**Comment by laurent22** [comment]
@personalizedrefrigerator, you're right that if change the transfer type, creating link via drag and drop no longer works. Changing it to "moveCopy" still works, but it also still has the "+" icon. I've also tried playing with custom icons but without any success. I guess that would be nice to fix this, but not critical, so moving it to backlog for now
